sdk/servicebus/service-bus/src/serviceBusClient.ts

```diff
@@ -154,6 +170,11 @@ export class ServiceBusClient {
   /**
    * Creates a receiver for an Azure Service Bus queue in peekLock mode.
    *
+   * In peekLock mode, the receiver has a lock on the message for the duration specified on the
+   * queue or subscription.
+   *
+   * If the message is not settled during this time it becomes available to other receivers.
```

Comment:
Is it worth mentioning that receiving a message too many times without settling causes it to end up in the dead letter queue? (Or am I wrong about that?)

Reply:
You're not wrong - I believe the same thing happens if you call abandon().

Let me think about that one a bit more.

Reply:
This ended up getting added :)